### DISCUSSION TITLE
How to reduce swiper bundle size in an Angular project

### DISCUSSION THREAD

**Discussion by marshall86** [nolimits4web/swiper#7438] [Q&A]
I'd like to reduce the bundle size of the Swiper integration in my Angular project. Analyzing the project bundle, I can see there are many ".mjs" files like zoom, mousewheel ecc that I don't need and I'd like to remove.

Is there a way to achieve this?
Below the integration using Swiper 11.0.6

`
import { register } from 'swiper/element/bundle';
import { Swiper, SwiperOptions } from 'swiper/types';
register();

@Component({.... })
export class MyComponent {
	swiperInstance: Swiper;
        sliderOptions: SwiperOptions = { ...sliderSettings.default, spaceBetween: 20, slidesPerView: 'auto' };

	@ViewChild('swiper') set swiper(swiperRef: ElementRef) {
		if (!swiperRef?.nativeElement) {
			return;
		}
		this.swiperInstance = swiperRef?.nativeElement.swiper;
		this.swiperInstance.updateSlides();
	}
}
`

`<swiper-container #swiper [spaceBetween]="sliderOptions.spaceBetween" [slidesPerView]="sliderOptions.slidesPerView">
	<swiper-slide>
		...
	</swiper-slide>
</swiper-container>`

**Comment by Gawel91** [comment]
Would love to reduce it too, on a Nuxt project
